junction/templates/proposals/detail/public_comments.html

```diff
@@ -4,79 +4,105 @@
+                                    </small>
+                                </div>
+                            </div>
+                        </div>
                     </div>
-                </div>
-                <form id="prosal-comment-up-vote" method="POST" action="{{ comment.get_up_vote_url }}">
-                    {% csrf_token %} {{ proposal_comment_vote_form }}
-                </form>
-                <form id="prosal-comment-down-vote" method="POST" action="{{ comment.get_down_vote_url }}">
-                    {% csrf_token %} {{ proposal_comment_vote_form }}
-                </form>
-            </div>
-        {% endif %}
-        <div class="bg-info">
-            {{ comment.comment|markdown }} <br/>
-            Commented by: <b>{{ comment.commenter }} </b> at {{ comment.created_at }}
-            <hr/>
-        </div>
-    {% endfor %}
+                {% endfor %}
+
+                <div class="add-comment-wrapper">
+                    {% if user.is_authenticated %}
+                        <a href="#" class="js-add-comment">add a comment</a>
+
+                        <form method="POST" class="new-comment js-new-comment-form" style="display:none;" action="{% url 'proposal-comment-create' proposal.conference.slug proposal.slug %}">
+                            {% csrf_token %}
+                            {{ proposal_comment_form.media }}
 
-    {% if user.is_authenticated %}
-        <form method="POST" action="{% url 'proposal-comment-create' proposal.conference.slug proposal.slug %}">
-            {% csrf_token %}
-            {{ proposal_comment_form.media }}
+                            {% bootstrap_field proposal_comment_form.comment %}
 
-            {% bootstrap_field proposal_comment_form.comment %}
+                            {% buttons %}
+                                <button type="submit" class="btn btn-primary">
+                                    Add Comment
+                                </button>
+                            {% endbuttons %}
+                        </form>
+                    {% else %}
+                        <p>
+                              <a href="/accounts/login/?next={{ proposal.get_absolute_url }}">
+                                    Login
+                              </a>
+                              to add a new comment.
+                        </p>
+                    {% endif %}
+                </div>
+            </div>
 
-            {% buttons %}
-            <button type="submit" class="btn btn-primary">
-                Add Comment
-            </button>
-            {% endbuttons %}
-        </form>
-        {% else %}
-        <p>
-              <a href="/accounts/login/?next={{ proposal.get_absolute_url }}">
-                    Login
-              </a>
-              to add a new comment.
-        </p>
-    {% endif %}
+            <div class="tab-pane" id="js-reviewers" role="tabpanel">
```

Comment:
Reviewers tab should be visible for reviewers only. In most of the cases viewers won't be logged in or non reviewers, so it is ok. We can hold on for time being, @sivaa is planning to create appropriate workflow for reviewers . I will merge this for now.